package.json

```diff
@@ -22,7 +22,9 @@
     "url": "https://github.com/turbo87/sidebar-v2.git"
   },
   "scripts": {
-    "test": "gulp lint"
+    "lint": "gulp lint",
+    "test": "jest",
+    "cover": "jest --coverage"
```

Comment:
we can just call `yarn test --coverage`, so I don't think the extra `cover` script is necessary :)

Reply:
Ok, good to know. I'll remove the `cover` target.

Reply:
I've removed the `cover` target and have replaced `npm` with `yarn`.

Reply:
... although I've just found out that `yarn` can't install `openlayers@^3`.  Odd that the installation works with `npm` though...